kubernetes/helm_charts/monitoring/ingestion-kafka-exporter/templates/servicemonitor.yaml

```diff
@@ -0,0 +1,33 @@
+{{- if .Values.prometheus.serviceMonitor.enabled }}
```

Comment:
@Kaali09 why are we having a separate chart? we were supposed to have a soft link right ?

Reply:
@rjshrjndrn we are using .Chart.Name variable in deployment manifest . chart.yaml different for both .

Reply:
please override it in values as `nameOverride` can override the value.